spec/javascripts/components/accordion-spec.js

```diff
@@ -7,40 +7,54 @@ describe('Accordion component', function () {
-          '<span class="govuk-accordion__show-all-text">Show all sections</span>' +
-        '</button>' +
-      '</div>' +
-      '<div class="govuk-accordion__section">' +
-        '<div class="govuk-accordion__section-header">' +
-          '<h2 class="govuk-accordion__section-heading">' +
-            '<span class="govuk-accordion__section-button" id="default-id-078ef791-heading-1">Writing well for the web</span>' +
-          '</h2>' +
-        '</div>' +
-        '<div id="default-id-078ef791-content-1" class="govuk-accordion__section-content" aria-labelledby="default-id-078ef791-heading-1">' +
-          '<p class="govuk-body">This is the content for Writing well for the web.</p>' +
-        '</div>' +
-      '</div>' +
-      '<div class="govuk-accordion__section">' +
-        '<div class="govuk-accordion__section-header">' +
-          '<h2 class="govuk-accordion__section-heading">' +
-            '<span class="govuk-accordion__section-button" id="default-id-078ef791-heading-2">Writing well for specialists</span>' +
-          '</h2>' +
-        '</div>' +
-        '<div id="default-id-078ef791-content-2" class="govuk-accordion__section-content" aria-labelledby="default-id-078ef791-heading-2">' +
-          '<p class="govuk-body">This is the content for Writing well for specialists.</p>' +
-        '</div>' +
-      '</div>' +
-    '</div>'
+  var html = `
+    <div class="gem-c-accordion" data-show-text="Show" data-hide-text="Hide" data-track-show-all-clicks="true" data-show-all-text="Show all sections" data-hide-all-text="Hide all sections" data-track-sections="true" data-this-section-visually-hidden=" this section">
+      <div class="govuk-accordion__controls">
+        <button type="button" class="govuk-accordion__show-all gem-c-accordion__show-all" aria-expanded="false">
+          <span class="govuk-accordion__show-all-text">Show all sections</span>
+        </button>
+      </div>
+      <div class="govuk-accordion__section">
+        <div class="govuk-accordion__section-header">
+          <h2 class="govuk-accordion__section-heading" data-track-count="accordionSection" data-track-options="{&quot;dimension114&quot;:1}">
+            <button type="button" class="govuk-accordion__section-button" aria-expanded="false">
+              <span class="govuk-accordion__section-heading-text"><span class="govuk-accordion__section-heading-text-focus">Accordion Section Heading Text</span></span>
+              <span class="govuk-visually-hidden govuk-accordion__section-heading-divider">, </span>
+                <span class="govuk-accordion__section-toggle" data-nosnippet="">
+                  <span class="govuk-accordion__section-toggle-focus">
+                    <span class="govuk-accordion-nav__chevron govuk-accordion-nav__chevron--down"></span>
+                    <span class="govuk-accordion__section-toggle-text">Show<span class="govuk-visually-hidden"> this section</span>
+                  </span>
```

Comment:
It looks like these additional lines are to make the markup closer to the actual rendered content, but is there a particular reason for this? I'd generally recommend that fixture markup is as simple as possible - if these extra spans are needed for a test then fair enough, but I can't see them being referenced anywhere.